api/src/test/java/com/epam/pipeline/controller/metadata/MetadataControllerTest.java

```diff
@@ -252,7 +252,7 @@ public void shouldUploadMetadataFromFile() throws Exception {
 
         verify(mockMetadataApiService).uploadMetadataFromFile(any(EntityVO.class),
                 multipartFileCaptor.capture(), eq(true));
-        assertRequestFile(multipartFileCaptor.getValue(), "file.txt", "file.txt".getBytes());
+        assertRequestFile(multipartFileCaptor.getValue());
```

Comment:
It doesn't seem like a good idea to hardcode a file name and its content. Let's pass these parameters explicitly.